src/pages/startedPage/startedPage.less

```diff
@@ -0,0 +1,66 @@
+.startedPage{
+    display: flex;
+    justify-content: center;
+    margin: 0 50px 80px 50px;
+}
+
+.description{
```

Comment:
As suggested by @LordAldi in our last development hour, we should put this CSS object inside its parent component so we can have better isolation and not affecting other pages' styling.